### DISCUSSION TITLE
Loading snapshots through config file without api server

### DISCUSSION THREAD

**Discussion by ShrirAm-cse** [firecracker-microvm/firecracker#3765] [Q&A] [answered]
Is there a way where i can load a snapshot from config file like - 
`./firecracker --no-api --config-file <path_to_the_configuration_file> `
where i specify the mem file and snapshot file paths in the config itself?

**Comment by kalyazin** [comment, marked as answer]
@ShrirAm-cse thanks for your question. No, this is currently not supported and we have not been planning to implement it, because it would probably require introducing a separate config file just for loading snapshots, to avoid ambiguity. At the moment we do not have a good use case to justify that effort.
Do you have a use case where it is beneficial?
